pkg/k8shandler/elasticsearch.go

```diff
@@ -434,3 +434,76 @@ func (req *ElasticsearchRequest) ListIndicesForAlias(aliasPattern string) ([]str
 	}
 	return response, nil
 }
+
+func AddAliasForOldIndices(clusterName, namespace string, client client.Client) (bool, error) {
+	// get .operations.*/_alias
+	// get project.*/_alias
+	/*
+		{
+			"project.test.107d38b1-413b-11ea-a2cd-0a3ee645943a.2020.01.27" : {
+				"aliases" : {
+					"test" : { }
+				}
+			},
+			"project.test2.8fe8b95e-4147-11ea-91e1-062a8c33f2ae.2020.01.27" : {
+				"aliases" : { }
+			}
+		}
+	*/
+
+	successful := true
+
+	payload := &esCurlStruct{
+		Method: http.MethodGet,
+		URI:    "project.*,.operations.*/_alias",
+	}
+
+	curlESService(clusterName, namespace, payload, client)
+
+	// alias name choice based on https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model
+	for index := range payload.ResponseBody {
+		// iterate over each index, if they have no aliases that match the new format
+		// then PUT the alias
+
+		indexAlias := ""
+		if strings.HasPrefix(index, "project.") {
```

Comment:
I presume the answer is no, but is there anyway to abstract this to rely on indexmanagement?  I don't believe there is anyway to interpret the new IM policy = alias old index with XXXXX

Reply:
do you have an example of a setting that it would look at?